homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration uses entity services.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: make sure flows end with created entry or abort
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: devices are setup asynchronously and marked as unavailable until they are ready.
+  unique-config-entry:
+    status: exempt
+    comment: The integration only supports a single config entry.
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: todo
+  reauthentication-flow: done
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info: done
+  discovery:
+    status: exempt
+    comment: There is no option to discover devices without adding the integration.
+  docs-data-update: todo
+  docs-examples: done
+  docs-known-limitations: done
+  docs-supported-devices: done
+  docs-supported-functions: done
+  docs-troubleshooting: todo
+  docs-use-cases: done
+  dynamic-devices: done
+  entity-category: done
+  entity-device-class: done
```

Comment:
```py
TIME_SENSORS: list[SensorEntityDescription] = [
    SensorEntityDescription(
        key="remaining_time",
        translation_key="remaining_time",
    ),
    SensorEntityDescription(
        key="auto_off_set",
        translation_key="auto_shutdown",
        entity_registry_enabled_default=False,
    ),
]
```
What's the unit of measurement?

Reply:
Both of this are strings and don't have a unit of measurement (e.g. "00:00:10")
Note that the `auto_shutdown` is never changed, but the "remaining_time" will probably moved to a timestamp sensor as it changes frequently (when I started it was changing once per minute, there are reports now of every second)

Reply:
These should then either become a timestamp or a duration device class

Reply:
Changed status to `todo` as this would need a breaking change